client/src/elements/schema-editor/schema-editor-array.html

```diff
@@ -4,6 +4,7 @@
   <legend>
     ${schema.title & toolT}
   </legend>
+  <div class="schema-editor-array__entry-border ${showNotifications ? 'schema-editor-notification-column' : ''}"></div>
```

Comment:
any chance to implement this without the extra element here? could this be done with a class on the `schema-editor-array__entry` and if necessary a pseudo-element?